sycl/include/CL/sycl/detail/defines_elementary.hpp

```diff
@@ -40,11 +40,7 @@
 
 #ifndef __SYCL_DEPRECATED
 #ifndef SYCL_DISABLE_DEPRECATION_WARNINGS
-#ifdef _WIN32
-#define __SYCL_DEPRECATED(message) __declspec(deprecated(message))
-#else
-#define __SYCL_DEPRECATED(message) __attribute__((deprecated(message)))
-#endif
+#define __SYCL_DEPRECATED(message) [[deprecated(message)]]
```

Comment:
This one is only available since C++14. Do we still want to support C++11? If so, this should be guraded with ifdefs.

Reply:
No we do not want to support C++11.